Remotion/Web/Development.WebTesting.IntegrationTests/TreeViewControlObjectTest.cs

```diff
@@ -88,7 +88,7 @@ public void TestNodeIsChecked ()
               .With.Message.EqualTo(
                   AssertionExceptionUtility.CreateExpectationException(
                       Driver,
-                      "The checkbox could not be found: Unable to find xpath: ./tbody/tr/td[a[contains(@onclick, 'TreeView_SelectNode')]]/input[@type='checkbox']").Message));
+                      "The checkbox could not be found: Unable to find xpath: ./tbody/tr/td[a[@data-inline-event-target]]/input[@type='checkbox']").Message));
```

Comment:
```suggestion
                      "The checkbox could not be found: Unable to find xpath: ./tbody/tr/td[a[contains(@data-event-content-onclick, 'TreeView_SelectNode')]]/input[@type='checkbox']").Message));
```

given we now have the attribute with the actual script body, we should go back to `contains` to preserve maximum compatiblity.